tests/Eccube/Tests/Web/Admin/Setting/Shop/CsvControllerTest.php

```diff
@@ -75,4 +76,78 @@ public function testSetCsv()
         $this->app['orm.em']->getConnection()->rollback();
     }
 
+    /**
+     * @expectedException Symfony\Component\HttpKernel\Exception\NotFoundHttpException
+     */
+    public function testRoutingCsvFail()
+    {
+        if ($this->app['config']['database']['driver'] == 'pdo_sqlite') {
+            // 何故か CsvType が EntityNotFoundException: Entity was not found. になる
+            $this->markTestSkipped('Can not support for sqlite3');
+        }
+        $this->client->request('GET', $this->app['url_generator']->generate('admin_setting_shop_csv', array('id' => 9999)));
+        $this->fail();
+    }
+
+    public function testSubmit()
+    {
+        if ($this->app['config']['database']['driver'] == 'pdo_sqlite') {
+            // 何故か CsvType が EntityNotFoundException: Entity was not found. になる
+            $this->markTestSkipped('Can not support for sqlite3');
+        }
+
+        $csvType = CsvType::CSV_TYPE_PRODUCT;
+        $CsvOut = $this->createCsv($csvType);
+        $CsvNotOut = $this->createCsv($csvType);
+
+        $form = array(
+            '_token' => 'dummy',
+            'csv_type' => $csvType,
+            'csv_not_output' => array(
+                $CsvOut->getId()
+            ),
+            'csv_output' => array(
+                $CsvNotOut->getId()
+            )
+        );
+
+        $this->client->request('POST',
+            $this->app['url_generator']->generate('admin_setting_shop_csv', array('id' => $csvType)),
```

Comment:
Please use `$this->app->url()` instead of `$this->app['url_generator']->generate()`